QEfficient/utils/_utils.py

```diff
@@ -128,6 +128,9 @@ def load_hf_tokenizer(model_name: str, cache_dir: Optional[str] = None, hf_token
     model_hf_path = hf_download(repo_id=model_name, cache_dir=cache_dir, allow_patterns=["*.json", "*.py", "*token*"])
     #FIXME(ochougul): should this always return left padded tokenizer?
     tokenizer = AutoTokenizer.from_pretrained(model_hf_path, padding_side=padding_side, trust_remote_code=True, **kwargs)
+    
+    padding_check_and_fix(tokenizer)
```

Comment:
Suggested by @ochougul to handle repetition of code.

Reply:
there is still repetition of function, we should ideally add check in `load_tokenizer` and use that and raise a `ValueError` in export only. and mention in doc to use `right `padding for LL api users.

Reply:
I think we should check and correct the tokenizer padding if user passes otherwise, as we already have a way to fix it.
So, when we load in our function, we always make sure we pad it the right way and also handle the `eos_token_id`,
while whenever tokenizer is executed at lowest level function, we again check that the tokenizer is correct if not we fix it inside that function.

This handles if user decides to use low level function, it will work irrespective of developer's ignorance.